AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -42,4 +49,60 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
   return [ASLayout layoutWithLayoutableObject:self size:constrainedSize.min];
 }
 
+- (ASLayoutSpec *)finalLayoutableWithParent:(ASLayoutSpec *)parentSpec;
+{
+  return nil;
```

Comment:
Please follow up on this after landing this initial version.  I'd like to aim for a solution that would expect / handle a return value of "self" here.